source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/deploy/reach/reach_env_cfg.py

```diff
@@ -153,7 +153,7 @@ class RewardsCfg:
     """Reward terms for the MDP."""
 
     end_effector_keypoint_tracking = RewTerm(
-        func=mdp.keypoint_command_error,
+        func=mdp.KeypointCommandError,
```

Comment:
Please follow Python style-guide. For functional-classes (only for MDP terms), we still use snake-case to not make a distinction.